src/program/temperature_pulse.cpp

```diff
@@ -220,8 +220,7 @@ void temperature_pulse(){
    // If local temperature is set then also initalise local temperatures
    if(sim::local_temperature==true){
       for(int mat=0;mat<mp::material.size();mat++){
-         if(mp::material[mat].couple_to_phonon_temperature==true) mp::material[mat].temperature=sim::TTTp;
-         else mp::material[mat].temperature=sim::TTTe;
```

Comment:
since u had set both sim::TTTe and TTTp to sim::temperature before the if  else,  TTTe equals to TTTp. That`s why I think it is useless

line 217:
> 	sim::TTTe=sim::temperature;
> 	sim::TTTp=sim::temperature;
>